doc/nrf/releases_and_maturity/releases/release-notes-2.4.0.rst

```diff
@@ -622,7 +622,7 @@ nRF9160 samples
 PMIC samples
 ------------
 
-* Added :ref:`npm1300_fuel_gauge` sample that demonstrates how to calculate the battery state of charge using the :ref:`nrfxlib:nrf_fuel_gauge`.
+* Added the nPM1300: Fuel gauge sample that demonstrates how to calculate the battery state of charge using the :ref:`nrfxlib:nrf_fuel_gauge`.
```

Comment:
```suggestion
* Added the :ref:`nPM1300: Fuel gauge <npm13xx_fuel_gauge>` sample that demonstrates how to calculate the battery state of charge using the :ref:`nrfxlib:nrf_fuel_gauge`.
```
You can use this formatting here and elsewhere to keep the link, but hide the updated title. Chronologically, this will be fine (1304 will be hidden for older notes), and the link will still work.